ext/ffi/00_ffi.js

```diff
@@ -326,6 +330,33 @@
             },
           );
         }
+
+        if (needsUnpacking && !isNonBlocking) {
+          const call = this.symbols[symbol];
+          const parameters = symbols[symbol].parameters;
+          const v = new Int32Array(2);
```

Comment:
Would it be a bad idea to have this be a general buffer for the entire dlopen, so that all calls use the retval memory? Definitely a minor optimisation memory optimisation, though.